> Hi @bernhardkaindl that to me looks like potentially `root` is not properly set up in the environment where the tests are run, because the `cppyy_backend` usually comes with that in our case. I will have a look into this.
> 
> Is this the complete output from the tests?

It's the only failure, so you could just disable it, but I agree it should be able to use the cppyy_backend which is in `root`, it seems to use it already, just fails to import the backend of it:
```py
Command: "/home/kaindlb/home/spack/.tree1/linux-ubuntu18.04-zen2/gcc-11.2.0/python-3.8.11-qh4rt3wnm2rcxqpnxqzfpnktht5d3wmy/bin/python" "-m" "unittest" "discover" "-s" "/home/kaindlb/home/spack/.stage/spack-stage-podio-0.13.2-23f4e7d22sqm5ry5ddvwumu3nwwecdkf/spack-src/python"
Directory: /home/kaindlb/home/spack/.stage/spack-stage-podio-0.13.2-23f4e7d22sqm5ry5ddvwumu3nwwecdkf/spack-build-23f4e7d/tests
ERROR: test_EventStore (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_EventStore
Traceback (most recent call last):
  File "/home/kaindlb/home/spack/.tree1/linux-ubuntu18.04-zen2/gcc-11.2.0/python-3.8.11-qh4rt3wnm2rcxqpnxqzfpnktht5d3wmy/lib/python3.8/unittest/loader.py", line 436, in _find_test_path
    module = self._get_module_from_name(name)
  File "/home/kaindlb/home/spack/.tree1/linux-ubuntu18.04-zen2/gcc-11.2.0/python-3.8.11-qh4rt3wnm2rcxqpnxqzfpnktht5d3wmy/lib/python3.8/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/home/kaindlb/home/spack/.stage/spack-stage-podio-0.13.2-23f4e7d22sqm5ry5ddvwumu3nwwecdkf/spack-src/python/test_EventStore.py", line 5, in <module>
    from EventStore import EventStore
  File "/home/kaindlb/home/spack/.stage/spack-stage-podio-0.13.2-23f4e7d22sqm5ry5ddvwumu3nwwecdkf/spack-src/python/EventStore.py", line 5, in <module>
    from ROOT import gSystem
  File "/home/kaindlb/home/spack/.tree1/linux-ubuntu18.04-zen2/gcc-11.2.0/root-6.24.06-65jied6ykiu5a75wug7bm2l254c525h5/lib/ROOT/__init__.py", line 22, in <module>
    import cppyy
  File "/home/kaindlb/home/spack/.tree1/linux-ubuntu18.04-zen2/gcc-11.2.0/root-6.24.06-65jied6ykiu5a75wug7bm2l254c525h5/lib/cppyy/__init__.py", line 131, in <module>
    from ._cpython_cppyy import *
  File "/home/kaindlb/home/spack/.tree1/linux-ubuntu18.04-zen2/gcc-11.2.0/root-6.24.06-65jied6ykiu5a75wug7bm2l254c525h5/lib/cppyy/_cpython_cppyy.py", line 18, in <module>
    c = loader.load_cpp_backend()
  File "/home/kaindlb/home/spack/.tree1/linux-ubuntu18.04-zen2/gcc-11.2.0/root-6.24.06-65jied6ykiu5a75wug7bm2l254c525h5/lib/cppyy_backend/loader.py", line 74, in load_cpp_backend
    raise RuntimeError("could not load cppyy_backend library")
```

--- a/var/spack/repos/builtin/packages/podio/package.py
+++ b/var/spack/repos/builtin/packages/podio/package.py
@@ -34,6 +34,7 @@ class Podio(CMakePackage):
     # cpack config throws an error on some systems
     patch('cpack.patch', when="@:0.10.0")
     patch('dictloading.patch', when="@0.10.0")
+    patch('python-tests.patch', when='@:0.14.0')
 
     depends_on('root@6.08.06: cxxstd=17')
 

--- a/var/spack/repos/builtin/packages/podio/package.py
+++ b/var/spack/repos/builtin/packages/podio/package.py
@@ -17,6 +17,7 @@ class Podio(CMakePackage):
     tags = ["hep", "key4hep"]
 
     version('master', branch='master')
+    version('0.14', sha256='47f99f1190dc71d6deb52a2b1831250515dbd5c9e0f263c3c8553ffc5b260dfb')
     version('0.13.2', sha256='645f6915ca6f34789157c0a9dc8b0e9ec901e019b96eb8a68fb39011602e92eb')
     version('0.13.1', sha256='2ae561c2a0e46c44245aa2098772374ad246c9fcb1956875c95c69c963501353')
     version('0.13', sha256='e9cbd4e25730003d3706ad82e28b15cb5bdc524a78b0a26e90b89ea852101498')

--- a/var/spack/repos/builtin/packages/podio/python-tests.patch
+++ b/var/spack/repos/builtin/packages/podio/python-tests.patch
@@ -1,0 +1,12 @@
+index 5b6e13e..ac9ccf5 100644
+--- a/tests/CMakeLists.txt
++++ b/tests/CMakeLists.txt
+@@ -106,7 +106,7 @@ endif()
+ add_test( NAME pyunittest COMMAND python -m unittest discover -s ${CMAKE_SOURCE_DIR}/python)
+ set_property(TEST pyunittest
+              PROPERTY ENVIRONMENT
+-                      LD_LIBRARY_PATH=${CMAKE_CURRENT_BINARY_DIR}:${CMAKE_BINARY_DIR}/src:$ENV{LD_LIBRARY_PATH}
++                      LD_LIBRARY_PATH=${CMAKE_CURRENT_BINARY_DIR}:${CMAKE_BINARY_DIR}/src:$<TARGET_FILE_DIR:ROOT::Tree>
+                       PYTHONPATH=${CMAKE_SOURCE_DIR}/python:$ENV{PYTHONPATH}
+                       ROOT_INCLUDE_PATH=${CMAKE_SOURCE_DIR}/tests/datamodel:${ROOT_INCLUDE_PATH})
+ set_property(TEST pyunittest PROPERTY DEPENDS write)

